blob/service_test.go

```diff
@@ -293,17 +298,17 @@ func TestBlobService_Get(t *testing.T) {
 			},
 		},
 		{
-			name: "get all not found",
+			name: "empty result and err when blobs were not found ",
 			doFn: func() (interface{}, error) {
-				namespace := share.Namespace(tmrand.Bytes(share.NamespaceSize))
-				return service.GetAll(ctx, 1, []share.Namespace{namespace})
+				nid, err := share.NewBlobNamespaceV0(tmrand.Bytes(appns.NamespaceVersionZeroIDSize))
+				require.NoError(t, err)
+				return service.GetAll(ctx, 1, []share.Namespace{nid})
 			},
 			expectedResult: func(i interface{}, err error) {
 				blobs, ok := i.([]*Blob)
 				require.True(t, ok)
 				assert.Empty(t, blobs)
-				require.Error(t, err)
-				require.ErrorIs(t, err, ErrBlobNotFound)
+				assert.Nil(t, err)
```

Comment:
Still not addressed https://github.com/celestiaorg/celestia-node/pull/3223#discussion_r1656891153

Reply:
wdym? I change `assert.Empty(t, err)` to `assert.Nil(t, err)`

Reply:
Ok, you meant to do the same with blobs.

Reply:
`assert.NoError(t, err)`